### ISSUE TITLE
Lampflats with '' and 'NaN' in both telescope_1 and telescope_2 extensions

### ISSUE THREAD

**Issue by gmbrandt** [LCOGT/banzai-nres#84]
On the newest version of banzai nres main (commit 6243c0be89eaf3857011e69448a1926c8ddc18e8 ), old data from TLV that used to reduce fine (e.g., tlvnrs04-fa18-20190528-0008-w00.fits.fz), now throws an error when being loaded in. This is because of the logic within  banzai_nres/frames.py. The error is as follows:

```
RemoteTraceback                           Traceback (most recent call last)
RemoteTraceback: 
Traceback (most recent call last):
  File "/home/gmbrandt/envs/nres/lib/python3.8/site-packages/astropy/coordinates/angle_utilities.py", line 343, in parse
    found_angle, found_unit = self._thread_local._parser.parse(
  File "/home/gmbrandt/envs/nres/lib/python3.8/site-packages/astropy/extern/ply/yacc.py", line 333, in parse
    return self.parseopt_notrack(input, lexer, debug, tracking, tokenfunc)
  File "/home/gmbrandt/envs/nres/lib/python3.8/site-packages/astropy/extern/ply/yacc.py", line 1063, in parseopt_notrack
    lookahead = get_token()     # Get the next token
  File "/home/gmbrandt/envs/nres/lib/python3.8/site-packages/astropy/extern/ply/lex.py", line 386, in token
    newtok = self.lexerrorf(tok)
  File "/home/gmbrandt/envs/nres/lib/python3.8/site-packages/astropy/coordinates/angle_utilities.py", line 164, in t_error
    raise ValueError(
ValueError: Invalid character at col 0

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.8/multiprocessing/pool.py", line 125, in worker
    result = (True, func(*args, **kwds))
  File "/usr/lib/python3.8/multiprocessing/pool.py", line 48, in mapstar
    return list(map(*args))
  File "<ipython-input-31-2a03b2fa75ca>", line 8, in __call__
    run_pipeline_stages([{'path': filename}], self.context)
  File "/home/gmbrandt/Documents/Repositories/banzai/banzai/utils/stage_utils.py", line 44, in run_pipeline_stages
    images = [frame_factory.open(image_path, runtime_context) for image_path in image_paths]
  File "/home/gmbrandt/Documents/Repositories/banzai/banzai/utils/stage_utils.py", line 44, in <listcomp>
    images = [frame_factory.open(image_path, runtime_context) for image_path in image_paths]
  File "/home/gmbrandt/Documents/Repositories/banzai-nres/banzai_nres/frames.py", line 390, in open
    image.ra = Angle(image[f'TELESCOPE_{telescope_num}'].meta[f'{ra_dec_keyword}RA'], units.hourangle).deg
  File "/home/gmbrandt/envs/nres/lib/python3.8/site-packages/astropy/coordinates/angles.py", line 121, in __new__
    angle, angle_unit = util.parse_angle(angle, unit)
  File "/home/gmbrandt/envs/nres/lib/python3.8/site-packages/astropy/coordinates/angle_utilities.py", line 437, in parse_angle
    return _AngleParser().parse(angle, unit, debug=debug)
  File "/home/gmbrandt/envs/nres/lib/python3.8/site-packages/astropy/coordinates/angle_utilities.py", line 347, in parse
    raise ValueError("{} in angle {!r}".format(
ValueError: Invalid character at col 0 in angle 'NaN'
```
This is because this file has '' and 'NaN' in the 'ra' keyword (and 'dec' keyword) of both the telescope_1 and telescope_2 extension header. Calling f1 that file, and f2 a normal lampflat that processes fine, we have:

```
In [8]: f1['Telescope_1'].header['ra']
Out[8]: 'NaN'
In [9]: f1['Telescope_2'].header['ra']
Out[9]: ''
In [10]: f2['Telescope_2'].header['ra']
Out[10]: ''
In [11]: f2['Telescope_1'].header['ra']
Out[11]: '03:49:57.923'
```
In other words, most lampflats have, for some reason, a real ra and dec coordinate stored in one of the telescope extensions. This causes banzai to generate a Angle() coordinate correctly, even though the telescope is not really looking at anything. So this appears to be that a bug in the output data product coming from the telescope is masking a bug in the data reduction pipeline.
f1 and f2 in this case are tlvnrs04-fa18-20190528-0008-w00.fits.fz and  tlvnrs04-fa18-20190509-0001-w00.fits.fz, respectively.

The branch https://github.com/LCOGT/banzai-nres/tree/fix/temporary_tlv_nan illustrates one fix to this problem.
